examples/kafka/v1beta1/scramsecretassociation.yaml

```
@@ -138,6 +140,70 @@ spec:
 
 ---
 
+apiVersion: secretsmanager.aws.upbound.io/v1beta1
+kind: SecretPolicy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: kafka/v1beta1/scramsecretassociation
+  labels:
+    testing.upbound.io/example-name: scram-secret-association
+  name: example1
+spec:
+  forProvider:
+    policy: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Sid": "AWSKafkaResourcePolicy",
+            "Effect": "Allow",
+            "Principal": {
+              "Service": "kafka.amazonaws.com"
+            },
+            "Action": "secretsmanager:getSecretValue",
+            "Resource": "*"
+          }
+        ]
+      }
+    region: us-east-2
+    secretArnSelector:
+      matchLabels:
+        testing.upbound.io/example-number: one
```

```suggestion
    secretArnRef:
      name: example1
```
Every field with a Selector also supports a Ref containing the metadata.name of a resource of the appropriate Kind. This way you don't need the extra labels.

---

Hm... It's a bit misleading, here I see secret id https://marketplace.upbound.io/providers/upbound/provider-aws-secretsmanager/v1.21.1/resources/secretsmanager.aws.upbound.io/SecretVersion/v1beta1, in secretpolicy I see secret arn... So, names of managed resources can be referenced and they will be resolved into arn, if I understand correct

---

I've replaced selector with secretArnRef in example